praetorian_cli/ui/console/commands/tools.py

```
@@ -99,24 +123,125 @@ def _cmd_run(self, args):
+
+        caps = {k: v for k, v in TOOL_ALIASES.items() if not v.get('agent') and k != 'secrets'}
+        if caps:
+            table2 = Table(title='Capabilities', border_style=self.colors['dim'])
+            table2.add_column('Capability', style=f'bold {self.colors["primary"]}', min_width=16)
+            table2.add_column('Target', style=self.colors['accent'])
+            table2.add_column('Description')
+            for name, info in sorted(caps.items()):
+                table2.add_row(name, info['target_type'], info['description'])
+            self.console.print(table2)
+
+        self.console.print(f'\n[dim]Usage: use <name> or <name> <target_key>[/dim]')
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)